src/coord/View.ts

```diff
@@ -127,12 +128,14 @@ class View extends Transformable implements CoordinateSystemMaster, CoordinateSy
     /**
      * Set center of view
      */
-    setCenter(centerCoord?: number[]): void {
+    setCenter(centerCoord?: (number | string)[], api?: ExtensionAPI): void {
```

Comment:
Could we change these two parameters to required here? Optional seems to be unnecessary an will be easy to have bugs.